app.js

```diff
@@ -0,0 +1,37 @@
+// Import function from other file 
+import { compareNumbers } from './function.js'; 
```

Comment:
you only need the curly braces here if there is more than one exported function on the page you are importing.  Also, your js is named "functions" not "function"  this is why your tests weren't running